it/uk/gov/hmrc/apiplatformadminapi/stubs/ThirdPartyOrchestratorConnectorStub.scala

```diff
@@ -38,4 +38,17 @@ trait ThirdPartyOrchestratorConnectorStub extends WireMockExtensions {
           )
       )
   }
+
+  object GetDeveloper {
```

Comment:
This object usually matches the name of the connector method, i.e. `GetBySessionId`